lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -296,7 +292,6 @@ static void dumpProfileInfo(const std::vector<KernelLaunch> &kernelLaunches) {
 }
 
 void OCLBackend::doForwardPass() {
-  std::vector<KernelLaunch> kernelLaunches;
```

Comment:
Why make this a member of the class? It seems to only be used in this function's scope and is cleared before exiting the function.

Reply:
The plan is to use it more for profiling and the like in the future. Thus it is convenient to have as a member of the class.